operator/src/test/java/org/keycloak/operator/testsuite/integration/RealmImportTest.java

```diff
@@ -193,4 +194,33 @@ public void testNotWorkingRealmImport() {
                 });
     }
 
+    @Test
+    public void testFailedRealmImportWhenKeycloakNotAvailable() {
+        // Arrange
+        Keycloak kc = getDefaultKeycloakDeployment();
+        kc.getSpec().setInstances(0);
+
+        // don't wait for Keycloak being available, since it has no instances
+        deployKeycloak(k8sclient, getDefaultKeycloakDeployment(), false);
```

Comment:
This should reference the kc instance created above.

Reply:
OMG, good catch @shawkins - fixed!